app/src/main/java/io/gnosis/safe/ui/transactions/TransactionView.kt

```diff
@@ -100,7 +101,8 @@ sealed class TransactionView(
         @DrawableRes val confirmationsIcon: Int,
         val nonce: String,
         val methodName: String?,
-        val addressInfo: AddressInfoData
+        val addressInfo: AddressInfoData,
+        val isCancellation: Boolean
```

Comment:
Just small comment here,

As we agreed later on there will be new type of transaction for rejection transaction, and in UI it's handled as new type so I recommend to create new model type instead of just flag.

Reply:
@biafra23 you could add Rejection and RejectionQueued to io.gnosis.safe.ui.transactions.TransactionView

Reply:
To be clear `Rejection` should look like this?

```
    @JsonClass(generateAdapter = true)
    data class Rejection(
        @Json(name = "to") val to: Solidity.Address,
        @Json(name = "toInfo") val toInfo: AddressInfo?,
        @Json(name = "dataSize") val dataSize: Int,
        @Json(name = "value") val value: BigInteger,
        @Json(name = "methodName") val methodName: String?
    ) : TransactionInfo(TransactionType.Rejection)
```

Why does it need to have `methodName`, `value` and `dataSize`?

Reply:
Should we really call it `Rejection` if the backend calls it `cancellation_transaction`?

Reply:
Oh wait. That json sample was probably the response from the transaction service. 

@jpalvarezl 
Where can I find documentation on the `Rejection` txInfo type?